toshi_hazard_store/model/hazard_models_pydantic.py

```diff
@@ -0,0 +1,44 @@
+"""the hazard metatdata models are pydantic for (de)serialisaiton as json."""
+
+from datetime import datetime, timezone
+
+from pydantic import BaseModel, Field
+
+
+class CompatibleHazardCalculation(BaseModel):
+    """Provides a unique identifier for compatible Hazard Calculations"""
+
+    model_config = dict(populate_by_name=True)
+
+    unique_id: str = Field(..., alias="unique_id")
+    notes: str | None = Field(None, alias="notes")
```

Comment:
```suggestion
    notes: str | None = Field(default=None, alias="notes")
```

Reply:
to mimic the pattern of the members below that use `default_factory` I would suggest using `default` keyword argument